atlassian_jwt_auth/frameworks/common/asap.py

```diff
@@ -11,6 +13,7 @@
 def _process_asap_token(request, backend, settings, verifier=None):
     """ Verifies an ASAP token, validates the claims, and returns an error
     response"""
+    logger = logging.getLogger('asap')
```

Comment:
Do we call `logging.basicConfig()` anywhere? There might be inconsistencies if it's not called early. See https://stackoverflow.com/questions/7016056/python-logging-not-outputting-anything/56144390#56144390